frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
```

Comment:
This seems dubious. Height is calculated with `calc using vh`.
- Subtracting a constant value say `120px` will make it `non-responsive` 
Better to use percentages here. :)